web/pdf_page_view.js

```diff
@@ -129,6 +129,13 @@ var PDFPageView = (function PDFPageViewClosure() {
       }
       div.removeAttribute('data-loaded');
 
+      var event = new CustomEvent('pageremoved', {
+        detail: { pageNumber: div.getAttribute('data-page-number') },
```

Comment:
Yes. The reason being that I completely forgot about ```this.id``` :)

I changed that locally.